packages/strapi-generate-new/lib/create-starter-project.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const execa = require('execa');
+const fs = require('fs-extra');
+const uuid = require('uuid/v4');
+
+module.exports = async function createStarterProject(scope) {
+  console.log('Creating a starter project.');
+
+  // Name of the repository
+  const repository = scope.starter;
+  const destination = scope.rootPath;
+  const destinationName = scope.name;
+
+  // Clone the starter repository
+  await execa('git', ['clone', repository, destination], {
+    stdio: 'inherit',
+  });
+
+  // Install dependencies thanks to the package.json inside the starter repo
+  await execa('yarn', ['setup'], {
+    stdio: 'inherit',
+    cwd: destination,
+  });
+
+  // Read the package.json inside the backend directory (strapi)
+  const packageJSON = await fs.readJSON(`${destination}/backend/package.json`);
```

Comment:
You should check if the file exists first and through an error with a lodding explaining the starter does'nt match the template